app/modules/asset_groups/models.py

```diff
@@ -883,7 +883,9 @@ class AssetGroup(db.Model, HoustonModel):
         cascade='all, delete',
     )
 
-    assets = db.relationship('Asset', back_populates='asset_group', order_by='Asset.guid')
```

Comment:
Any reason for this?. I keep meaning to remove that as I don't think it's actually used but not looked into it properly to be sure